File: examples/poodle-surf/examples/ios-objc/PoodleSurfObjC/Generic Views/GradientView.m
//
//  GradientView.m
//  PoodleSurfObjC
//
//  Created by Westin Newell on 5/10/19.
//  Copyright © 2019 Haiku. All rights reserved.
//

#import "GradientView.h"

NS_ASSUME_NONNULL_BEGIN

@implementation GradientView

- (CAGradientLayer *)gradientLayer {
    return (CAGradientLayer *)self.layer;
}

+ (Class)layerClass {
    return CAGradientLayer.class;
}

@end

NS_ASSUME_NONNULL_END

File: examples/poodle-surf/examples/ios/PoodleSurf/Generic Views/GradientView.swift
//
//  GradientView.swift
//  PoodleSurf
//
//  Created by Westin Newell on 4/11/19.
//  Copyright © 2019 Haiku. All rights reserved.
//

import UIKit

class GradientView: UIView {
    var gradientLayer: CAGradientLayer { return layer as! CAGradientLayer }
    override class var layerClass: AnyClass { return CAGradientLayer.self }
}
